Policies/Kubernetes/README.md

```diff
@@ -0,0 +1,23 @@
+# Kubernetes Polices
+Here are sample files for building Azure Policy for Kubernetes policy defintions.
+
+> [!NOTE]
+> 
+> Azure Policy for Kubernetes is in Preview and only supports built-in policy definitions at this time.
+
+To onboard to Azure policy for AKS, please refer:
```

Comment:
Can you use Azure Policy for Kubernetes instead of Azure Policy for AKS as we support AKS Engine & Arc as well?